wizard/src/utils/file_generation.ts

```diff
@@ -36,16 +37,33 @@ export function generateSvMCF(csv: CsvData, mappings: Mapping): string {
   return "";
 }
 
+// convert Mapping object to a regular javascript object so it can be converted
+// into a JSON string
+function mappingToObject(mapping: Mapping): Record<string, MappingVal> {
+  if (_.isEmpty(mapping)) {
+    return {};
+  }
+  const obj = {};
```

Comment:
nit: shouldn't this technically be "let obj = {};" ?

Reply:
no, it will always be the same object, we're just adding entries to it. It's just like a map where the map object doesn't change, we're just setting key value pairs.

Reply:
Ah I guess you can change/update members of a const object. Weird :)